### DISCUSSION TITLE
link up efforts with Iris team

### DISCUSSION THREAD

**Discussion by dcherian** [pangeo-data/xESMF#184] [Ideas]
I just chatted with @bjlittle who pointed me to https://github.com/SciTools-incubator/iris-esmf-regrid

It seems like we should explore a merger of the two efforts. It'd be nice to have a single dask-aware ESMF wrapper that can be used with both Xarray and Iris since both communities are solving the same problems.

Are you having regular maintainer meetings where we can take this conversation further?

**Comment by huard** [comment]
@dcherian Good idea !

We used to, but there hasn't been one in a while. 

@raphaeldussin @stefraynaud @sol1105 Are you around over the next two weeks ?

**Comment by raphaeldussin** [comment]
@huard I'm around

**Comment by huard** [comment]
@raphaeldussin @stefraynaud @sol1105 @dcherian @bjlittle 

Please fill this poll: https://www.when2meet.com/?16178590-HJ6E6 before Thursday.

**Comment by bjlittle** [comment]
Hey guys, thanks for setting this up. Apologies for the tardy response 😄 

I just spoke to the lead developer on [iris-esmf-regrid](https://github.com/SciTools-incubator/iris-esmf-regrid) @stephenworsley and we both filled out the poll.

Hope we can all manage to hangout next week. Cheers! 👍

**Comment by dcherian** [comment]
TO help with scheduling, I don't think I really have to attend this meeting. It'd be nice to prioritize more xesmf maintainers attending.

**Comment by bjlittle** [comment]
Thanks for organising the meeting @huard

I'll forward the invite on to @stephenworsley 👍

**Comment by huard** [comment]
Had a nice meeting, next steps are to 
1. identify code that's common to both xESMF and Iris;
2. discuss where it should live.

**Reply by dcherian** [reply]
Nice! Sorry I couldn't make it. An important 5 hour meeting (!) got scheduled at the same time.

Are there big differences in the approach to wrapping ESMF?
